From 99574be5daa48b28ca8240ffe9790787e63c3ebf Mon Sep 17 00:00:00 2001
From: Salvatore Bonaccorso <carnil@debian.org>
Date: Sun, 28 Jun 2020 20:19:55 +0200
Subject: [PATCH 1/2] systemd/Makefile: Drop exlicit setting of unit_dir

The used variable is named unitdir in configure.ac and
systemd/Makefile.am otherwise but is used in a single place as unit_dir.

The setting has no effect, but if later commits would use a base to the
systemd unit files directory for installing further files this would
void the possibility to explicitly set a systemd unit files directory
via configure with --with-systemd=unit-dir-path.

Fixes: 0fbf91a4fd90 ("Include systemd unit files in "dist" and "install".")
Signed-off-by: Salvatore Bonaccorso <carnil@debian.org>
---
 systemd/Makefile.am | 1 -
 1 file changed, 1 deletion(-)

diff --git a/systemd/Makefile.am b/systemd/Makefile.am
index 75cdd9f54c81..f089c8831902 100644
--- a/systemd/Makefile.am
+++ b/systemd/Makefile.am
@@ -45,7 +45,6 @@ man5_MANS	= nfs.conf.man
 man7_MANS	= nfs.systemd.man
 EXTRA_DIST = $(unit_files) $(man5_MANS) $(man7_MANS)
 
-unit_dir = /usr/lib/systemd/system
 generator_dir = /usr/lib/systemd/system-generators
 
 EXTRA_PROGRAMS	= nfs-server-generator rpc-pipefs-generator
-- 
2.27.0

